src/_scss/pages/homepageUpdate/_wordOfTheDay.scss

```diff
@@ -104,5 +107,24 @@
 
   .word-of-the-day__error-message {
     text-align: center;
+    color: $gray-2;
+  }
+  .word-of-the-day-paragraph-one{
+    font-size: rem(24);
```

Comment:
font size variables from _scss/core/variables.scss can be used here and in the word-of-the-day-paragraph-two class below

Reply:
changed to font size vars. I changed the other paragraph as well.